packages/design-tokens/LICENSE

```diff
@@ -1,6 +1,6 @@
 MIT License
```

Comment:
Aligning LICENSE with packages so it passes `yarn constraints`. I've created an issue to update our licenses across the monorepo to align with legal here https://github.com/MetaMask/metamask-design-system/issues/130